test/integration/page-objects/catalog.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+var h = require('../helpers.js');
+var extend = require('../helpers/extend').extend;
+var inputs = require('../helpers/inputs');
+var Page = require('./page').Page;
+//var logger = require('../helpers/logger');
+
+var AddTemplateModal = function(project) {
+  this.project = project;
+  this.modal = element(by.css('.modal-dialog'));
+  this.checkboxes = this.modal.all(by.css('input[type="checkbox"]'));
+  this.processBox = this.checkboxes.get(0);
+  this.saveBox = this.checkboxes.get(1);
+  this.continue = this.modal.element(by.cssContainingText('.btn-primary', 'Continue'));
+  this.cancel = this.modal.element(by.cssContainingText('.btn-default', 'Cancel'));
+  this.process = function() {
+    inputs.check(this.processBox);
+    inputs.unCheck(this.saveBox);
+    this.continue.click();
+    return browser
+            .sleep(500)
+            .then(function() {
+              // lazy require to avoid potential of circular dependencies
+              var CreateFromTemplatePage = require('./createFromTemplate').CreateFromTemplatePage;
+              return new CreateFromTemplatePage(this.project);
+            }.bind(this));
+  };
+  this.save = function() {
+    inputs.unCheck(this.processBox);
+    inputs.check(this.saveBox);
+    this.cancel.click();
+    browser.sleep(500);
+    return this;
+  };
+};
+
+var CatalogPage = function(project) {
+  this.project = project;
+};
+
+extend(CatalogPage.prototype, Page.prototype, {
+  getUrl: function() {
+    // ?tab=tab=fromFile, ?tab=fromCatalog, ?tab=deployImage
+    return 'project/' + this.project.name + '/create';
+  },
+  _findTabs: function() {
+    var tabs = element(by.css('.nav-tabs'));
+    h.waitForElem(tabs);
+    return tabs;
+  },
```

Comment:
Suggest a utility for clicking tabs that we can use for any page. We have a lot of tabs.